react/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "pingone-customers-sample-custom-signon-client",
```

Comment:
Does the name need to change?

Reply:
Yup, since we have `pingone-customers-sample-custom-signon` name in [https://github.com/pingidentity/pingone-customers-sample-custom-signon app](https://github.com/pingidentity/pingone-customers-sample-custom-signon/blob/2c04e74ed5925b22e2a23e8786c2e86aee3e4c7c/react/package.json#L2)